src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/multiples/UpdateReferralMultiplesController.java

```diff
@@ -0,0 +1,167 @@
+import uk.gov.hmcts.et.common.model.multiples.MultipleRequest;
+import uk.gov.hmcts.ethos.replacement.docmosis.helpers.ReferralHelper;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.CaseLookupService;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.DocumentManagementService;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.ReferralService;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.UserIdamService;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.CallbackRespHelper.multipleResponse;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.ReferralHelper.clearReferralDataFromCaseData;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.ReferralHelper.getNearestHearingToReferral;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.ReferralHelper.updateReferral;
+
+/**
+ * REST controller for the Update Referral event pages, formats data appropriately for rendering on the front end.
+ */
+@Slf4j
+@RequestMapping("/multiples/updateReferral")
+@RestController
+@RequiredArgsConstructor
+@SuppressWarnings({"PMD.UnnecessaryAnnotationValueElement", "PMD.ExcessiveImports"})
+public class UpdateReferralMultiplesController {
+    private final UserIdamService userIdamService;
+    private final ReferralService referralService;
+    private final DocumentManagementService documentManagementService;
+    private final CaseLookupService caseLookupService;
+    private static final String LOG_MESSAGE = "received update referral request for case reference : ";
+
+    /**
+     * Called for the first page of the Update Referral event.
+     * Populates the Referral hearing detail's section on the page.
+     * @param ccdRequest holds the request and case data
+     * @param userToken  used for authorization
+     * @return Callback response entity with case data and errors attached.
+     */
+    @PostMapping(value = "/aboutToStart", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "initialize data for referral update")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Accessed successfully",
+            content = {
+                @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = MultipleCallbackResponse.class))
+            }),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<MultipleCallbackResponse> updateReferralAboutToStart(
+        @RequestBody MultipleRequest ccdRequest,
+        @RequestHeader(value = "Authorization") String userToken) throws IOException {
+        log.info("ABOUT TO START UPDATE REFERRAL ---> " + LOG_MESSAGE + "{}", ccdRequest.getCaseDetails().getCaseId());
```

Comment:
Would it be possible to differentiate the Log messages used here from the log messages used on Singles?
Just in case users are reading logs in the future might make it easier?